### PR TITLE
Implement MySQL Client

### PR DESCRIPTION
**Pull request by GuyWhoCode**
Addresses #6

### CODE REVIEW

**Comment by GuyWhoCode** [comment]
ESLint linting issues originate from not using the SQL functions. Testing functions are in place to resolve said issues but are commented out because of the requirement to install MySQL server beforehand. Failure to install the MySQL server results in a SQL error stating that the localhost database cannot be connected to, causing the program to exit. 

The functions in question are `profSearch`, `updateProf`, and `addProf`.

**Review by ZombiMigz — CHANGES_REQUESTED** [review]
merge from dev to fix the .tsconfig linting error

File: api/server.ts [hunk 1]
```diff
@@ -10,3 +11,4 @@ app.post('/', (req, res) => {
 });
```
Comment (drjonah) [inline api/server.ts:60]: This is unnecessary but maybe instead of sending 'hello world', send a short message like 'connected' or something.

**Review by drjonah — APPROVED** [review]
server.ts
Check out my comment there. It is not urgent.

sql.ts
I would remove the test code at the bottom. For now, the comments are good, but we don't want those types of comments in dev.

Overall, the code looks good to me.

Wait! One more thing, make sure to resolve the ESLint errors.

File: api/server.ts (hunk 1 above)
Comment (GuyWhoCode) [reply api/server.ts:60]: This section is just a placeholder for the root / base endpoint, which is unrelated to my issue.

**Review by ZombiMigz — APPROVED** [review]

**Pull request merged**